src/components/organize/tasks/TaskAssigneesList.tsx

```diff
@@ -0,0 +1,42 @@
+import { useIntl } from 'react-intl';
+import { Box, Card, Grid } from '@material-ui/core';
+
+import { ZetkinAssignedTask } from 'types/tasks';
+import ZetkinPerson from 'components/ZetkinPerson';
+import ZetkinRelativeTime from 'components/ZetkinRelativeTime';
+import ZetkinSection from 'components/ZetkinSection';
+
+const TaskAssigneesList: React.FunctionComponent<{
+    assignedTasks: ZetkinAssignedTask[];
+}> = ({ assignedTasks }) => {
+    const intl = useIntl();
+
+    return (
+        <ZetkinSection title={ intl.formatMessage({ id: 'misc.tasks.taskAssigneesList.title' }, { numPeople: assignedTasks.length }) }>
+            <Grid container spacing={ 4 }>
+                { assignedTasks.map(task => {
+                    const completedText = task.completed ?
+                        // Relative completed time
+                        intl.formatMessage({
+                            id: 'misc.tasks.taskAssigneesList.completedStates.completed',
+                        }, {
+                            time:  <ZetkinRelativeTime datetime={ task.completed as string } />,
```

Comment:
This is displaying the wrong time because of timezones. Filing separate bug report as #366